maintainers/maintainer-list.nix

```diff
@@ -4769,6 +4769,11 @@
     githubId = 345808;
     name = "Jakub Okoński";
   };
+  FaustXVI = {
+    name = "Xavier Detant";
+    github = "FaustXVI";
+    githubId = 1016863;
```

Comment:
Ideally we should have any other method than github to reach out to you.

Reply:
Is my github anonymous email ok ?